checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/validation/AccessCheckoutCardValidatorTest.kt

```diff
@@ -73,4 +73,17 @@ class AccessCheckoutCardValidatorTest {
     fun `given a validator is constructed then should be able to fetch card configuration`() {
         assertEquals(cardConfiguration, accessCheckoutCardValidator.cardConfiguration)
     }
+
+    @Test
+    fun `given null cardConfiguration Will partially validate a luhn valid card`() {
+
+        val exp = Pair(ValidationResult(partial = true, complete = false), null)
+
+        accessCheckoutCardValidator = AccessCheckoutCardValidator(null)
+
+        val validationResult = accessCheckoutCardValidator.validatePAN("4111111111111112")
```

Comment:
This is actually a luhn invalid card, which is why it's not a `complete`ly valid card. Can you update the test description.

Reply:
🎯